sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -139,13 +139,11 @@ public String getEntityPath() {
      * Sends a message to a Service Bus queue or topic.
      *
      * @param message Message to be sent to Service Bus queue or topic.
-     *
      * @return The {@link Mono} the finishes this operation on service bus resource.
-     *
      * @throws NullPointerException if {@code message} is {@code null}.
      * @throws IllegalStateException if sender is already disposed.
      * @throws ServiceBusException if {@code message} is larger than the maximum allowed size of a single message or
-     *      the message could not be sent.
+     *                               the message could not be sent.
```

Comment:
Before this gets lost, please don't format existing lines. It adds noise to the PR.
Also, we do not align javadoc comments to the parameter names.

Reply:
According to your comment, the format of the existing line has been removed